ci/nodepool/nodepool.yaml

```diff
@@ -1,67 +1,88 @@
 script-dir: /home/pulp/pulp_packaging/ci/nodepool/scripts/
 
+# empty dir, stops nodepool from throwing a traceback on start
+images-dir: /etc/nodepool/images
+
 dburi: 'postgresql:///nodepool'
 
 zmq-publishers:
-  - tcp://<jenkins-master-hostname>:8881
+  - tcp://pulp-jenkins.rhev-ci-vms.eng.rdu2.redhat.com:8881
```

Comment:
I'm happy to leave this redacted, but we spam this all over github and IRC already, so I'm not sure it's worth keeping it hidden here.